app/models/activity.rb

```diff
@@ -42,8 +45,13 @@ class Activity < ApplicationRecord
     end
   }
 
+  scope :exclude_combined_children, -> {
+    where(combined_activity_parent: nil)
+  }
+
   scope :for_time_confirmed, ->(period_begin, period_end) {
                                accompaniment_eligible
+                                 .exclude_combined_children
```

Comment:
This would be the only change to the existing execution paths. We would not render activities which have been combined with a parent activity. The parent activity represents the group.